pokemon-app/src/components/MovesList.js

```diff
@@ -47,7 +53,15 @@ const MovesList = ({ pokemonMoves, hasEnemy, statsOnTop, isEnemyPokemon }) => {
       >
         <ol id={`${isEnemyPokemon ? "enemy-" : ""}moves-list`}>
           {pokemonMoves.map((moveData, moveIndex) => (
-            <li key={moveIndex}>{moveData.move.name}</li>
+            <li key={moveIndex}>
+              {moveData.move.name}
+
+              {effectivenessArray && effectivenessArray.length && (
```

Comment:
You should be able to shorten this to `effectivenessArray?.length && (`

Reply:
It's because the `MovesList` component is in both `SinglePokemon` and `EnemyPokemon` components. Since `SinglePokemon` doesn't render that array, I was worried that it would throw me an error since I wouldn't pass anything to that prop for `SinglePokemon`. Unless it wouldn't throw an error?